pygeoapi/provider/xarray_.py

```diff
@@ -554,11 +555,16 @@ def _to_datetime_string(datetime_obj):
     :returns: str representation of datetime
     """
 
+    LOGGER.warning(f"datetime value {datetime_obj}")
+    LOGGER.warning(f"datetime type {type(datetime_obj)}")
```

Comment:
Do these LOGGER warnings need to be present? Maybe remainders from debugging? Also double quoted str.

Reply:
No, they are for debugging purposes ATM. Will remove once tests pass in this PR.